internal/api/router.go

```diff
@@ -130,8 +130,18 @@ func NewRouter(handlers Handlers, cfg *config.Configuration, logger *logger.Logg
 			price.GET("/:id", handlers.Price.GetPrice)
 			price.PUT("/:id", handlers.Price.UpdatePrice)
 			price.DELETE("/:id", handlers.Price.DeletePrice)
+			price.POST("/unit", handlers.Price.CreatePriceWithUnitConfig)
 		}
 
+		priceUnit := v1Private.Group("/pricing/units")
```

Comment:
move this under prices routes under `prices/units
```suggestion
		priceUnit := v1Private.Group("/prices/units")
```